pepper-apis/config/application.conf.ctmpl

```diff
@@ -12,6 +12,7 @@
 
 {
     "dbUrl": "{{$conf.Data.dbUrl}}",
+    "defaultTimezone": "{{$conf.Data.defaultTimezone}}",
```

Comment:
We can just hardcode this--it's not a secret, so just dumping it in here as `"defaultTimezone": "UTC"` will simplify things.

Reply:
Reason I put the timezone in Vault is that it has to match what is in the dbUrl and seemed more likely that they remained in sync if values lived close together.
Can undo if you don't agree with rationale.

Reply:
Makes  sense to keep them closer together!

Reply:
As part of this PR, we should update *all* vault environments to make sure we're ready for this rollout.